_docs/_latest/setup/index.md

```diff
@@ -12,22 +12,30 @@ Platform (GCP) environments.
 
 ---
 
-| **[Installing Forseti Security]({% link _docs/latest/setup/install.md %})** |
-| :---------------------------------------------------------------------------- |
-| Install Forseti Security to help protect your GCP environments by monitoring your GCP resources to ensure that role-based access controls are set as you intended. The setup wizard automatically determines setup information, generates a deployment template, and creates a Forseti deployment. |
+**[Installing Forseti Security]({% link _docs/latest/setup/install.md %})**
 
-| **[Upgrading Forseti]({% link _docs/latest/setup/upgrade.md %})** |
-| :---------------------------------------------------------------------------- |
-| Get new features and update your GCP deployment by upgrading your Forseti Security installation to the latest version. |
+Install Forseti Security to help protect your GCP environments by monitoring your GCP resources to
+ensure that role-based access controls are set as you intended. The setup wizard automatically
+determines setup information, generates a deployment template, and creates a Forseti deployment.
 
-| **[Inventory]({% link _docs/latest/configure/inventory/index.md %})** |
-| :---------------------------------------------------------------------------- |
-| Set up Inventory to collect and store information about your GCP resources. Inventory helps you undersand your resources and take action to conserve resources, reduce cost, and minimize security exposure. |
+**[Upgrading Forseti]({% link _docs/latest/setup/upgrade.md %})**
 
-| **[Scanner]({% link _docs/latest/configure/scanner/index.md %})** |
-| :---------------------------------------------------------------------------- |
-| Set up Scanner to monitor your GCP resources for rule violations. Scanner uses the information from Inventory to regularly compare role-based access policies for your resources. |
+Get new features and update your GCP deployment by upgrading your Forseti Security installation
+to the latest version.
 
-| **[Enforcer]({% link _docs/latest/configure/enforcer/index.md %})** |
-| :---------------------------------------------------------------------------- |
-| Set up Enforcer to automatically correct policy discrepancies. Enforcer uses policies you create to compare the current state of your Compute Engine firewall to the desired state and uses Google Cloud APIs to make changes if it finds any differences. |
+**[Inventory]({% link _docs/latest/configure/inventory/index.md %})**
```

Comment:
Not sure if this is intended but the content is duplicated, the summary of Inventory, Scanner and Enforcer are already in _docs/_latest/configure/index.md

Reply:
fixed, thanks